tests/web/rpc/v1/test_endpoint_get_trace.py

```diff
@@ -35,6 +37,19 @@
 ) - timedelta(minutes=180)
 _SPAN_COUNT = 120
 _REQUEST_ID = uuid.uuid4().hex
+_NORMALIZED_FIELDS_TO_EXCLUDE = [
+    "retention_days",
+    "sign",
+    "attr_str",
+    "attr_num",
+    "span_id",
+    "timestamp",
+]
+_NORMALIZED_FIELDS_TO_INCLUDE = [
+    col.name
+    for col in get_entity(EntityKey("eap_spans")).get_data_model().columns
+    if col.name not in _NORMALIZED_FIELDS_TO_EXCLUDE
+]
```

Comment:
you are repeating the same logic in your functionality and in your test, it should only be specified once